common/models/role.js

```diff
@@ -213,15 +214,21 @@ module.exports = function(Role) {
    * @param {*} modelId The model ID
    * @param {*} userId The user ID
    * @param {String} principalType The user principalType (optional)
+   * @options {Object} options
+   * @property {accessToken} The access token used to authorize the current user.
    * @callback {Function} [callback] The callback function
    * @param {String|Error} err The error string or object
    * @param {Boolean} isOwner True if the user is an owner.
    * @promise
    */
-  Role.isOwner = function isOwner(modelClass, modelId, userId, principalType, callback) {
-    if (!callback && typeof principalType === 'function') {
+  Role.isOwner = function isOwner(modelClass, modelId, userId, principalType, options, callback) {
+    if (!callback && typeof options === 'function') {
+      callback = options;
+      options = {};
```

Comment:
This branch is not covered by any test, coveralls reports "Coverage decreased (-0.04%) to 89.558%" as a result. I think we can ignore that, but please let me know if you disagree! 

(The solution is to add a test that explicitly calls `Role.isOwner(modelClass, modelId, userId, principalType, callback)`.)